clang/bindings/python/clang/cindex.py

```diff
@@ -1722,23 +2300,23 @@ def location(self):
         return self._loc
 
     @property
-    def linkage(self):
+    def linkage(self) -> LinkageKind:
         """Return the linkage of this cursor."""
         if not hasattr(self, "_linkage"):
             self._linkage = conf.lib.clang_getCursorLinkage(self)
 
         return LinkageKind.from_id(self._linkage)
 
     @property
-    def tls_kind(self):
+    def tls_kind(self) -> TLSKind:
         """Return the thread-local storage (TLS) kind of this cursor."""
         if not hasattr(self, "_tls_kind"):
             self._tls_kind = conf.lib.clang_getCursorTLSKind(self)
```

Comment:
The `conf.lib.clang_getCursorTLSKind()` call leads to segmentation faults on my system (even before I changed anything) so I wasn't able to test this.